### ISSUE TITLE
Failed to activate the toggle-quotes package

### ISSUE THREAD

**Issue by ryanmclaughlin** [atom/toggle-quotes#30]
1. Installed toggle-quotes package.
2. Use shortcut Shift + Cmd + '

**Atom Version**: 1.0.5
**System**: Mac OS X 10.11
**Thrown From**: [toggle-quotes](https://github.com/atom/toggle-quotes) package, v0.11.2

### Stack Trace

Failed to activate the toggle-quotes package

```
At Cannot find module 'toggle-quotes'

Error: Cannot find module 'toggle-quotes'
  at Module._resolveFilename (module.js:328:15)
  at Function.Module._resolveFilename (/Applications/Atom.app/Contents/Resources/app.asar/src/module-cache.js:383:52)
  at Function.Module._load (module.js:270:25)
  at Module.require (module.js:357:17)
  at require (module.js:376:17)
  at Object.<anonymous> (/Users/DeGrasse/Dropbox (Personal)/Apps/Atom/.atom/packages/toggle-quotes/lib/main.coffee:1:18)
  at Object.<anonymous> (/Users/DeGrasse/Dropbox (Personal)/Apps/Atom/.atom/packages/toggle-quotes/lib/main.coffee:1:1)
  at Module._compile (module.js:452:26)
  at Object.requireCoffeeScript (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/coffee-cash/lib/coffee-cash.js:85:19)
  at Module.load (module.js:347:32)
  at Function.Module._load (module.js:302:12)
  at Module.require (module.js:357:17)
  at require (module.js:376:17)
  at Package.module.exports.Package.requireMainModule (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:759:34)
  at Package.module.exports.Package.activateConfig (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:261:14)
  at Package.module.exports.Package.activateNow (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:239:14)
  at /Applications/Atom.app/Contents/Resources/app.asar/src/package.js:836:25
  at Emitter.module.exports.Emitter.emit (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/event-kit/lib/emitter.js:82:11)
  at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:222:20)
  at /Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:3:61
  at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:524:16)
  at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:347:22)
  at HTMLDocument.module.exports.WindowEventHandler.onKeydown (/Applications/Atom.app/Contents/Resources/app.asar/src/window-event-handler.js:177:20)

```

### Commands

```
     -0:42.8.0 application:new-file (div.settings-view.pane-item)
     -0:35.4.0 emmet:remove-tag (atom-text-editor.editor.is-focused)
     -0:29 toggle-quotes:toggle (atom-text-editor.editor.is-focused)
     -0:19.9.0 editor:consolidate-selections (atom-text-editor.editor.is-focused)
     -0:19.9.0 core:cancel (atom-text-editor.editor.is-focused)
     -0:18.9.0 core:close (atom-text-editor.editor.is-focused)
     -0:17.4.0 pane:show-item-1 (div.settings-view.pane-item)
     -0:13.4.0 core:confirm (atom-text-editor.editor.mini.is-focused)
     -0:02.9.0 application:new-file (atom-workspace.workspace.scrollbars-visible-when-scrolling.theme-one-dark-syntax.theme-atom-material-ui)
     -0:00.8.0 toggle-quotes:toggle (atom-text-editor.editor.is-focused)
```

### Config

```json
{
  "core": {
    "ignoredNames": [
      ".git",
      ".svn",
      ".DS_Store"
    ],
    "themes": [
      "atom-material-ui",
      "one-dark-syntax"
    ]
  }
}
```

### Installed Packages

```coffee
# User
atom-material-ui, v0.5.13
bezier-curve-editor, v0.7.2
color-picker, v2.0.11
emmet, v2.3.12
file-icons, v1.6.2
linter, v1.3.2
linter-scss-lint, v2.0.0
meteor-api, v2.20.0
minimap, v4.12.2
minimap-find-and-replace, v4.2.0
minimap-selection, v4.3.0
pigments, v0.9.3
react, v0.12.6
toggle-quotes, v0.11.2

# Dev
No dev packages
```

**Comment by tloszko** [comment]
@ryanmclaughlin quick fix:

You need to replace first line of lib/main.coffee with:
```
{toggleQuotes} = require './toggle-quotes'
```

**Comment by izuzak** [comment]
Thanks for catching that, @tloszko :+1: -- would you like to submit a pull request to patch that up? 

And thanks for the report, @ryanmclaughlin. :+1:

**Comment by mnquintana** [comment]
Well that's an embarrassing typo :grimacing: - it's fixed now in 0.11.3. Sorry about that!
